monai/bundle/scripts.py

```diff
@@ -415,12 +419,22 @@ def ckpt_export(
     # convert to TorchScript model and save with meta data, config content
     net = convert_to_torchscript(model=net)
 
+    extra_files: Dict = {}
+    for i in ensure_tuple(config_file_):
+        # split the filename and directory
+        _, filename = os.path.split(i)
+        # remove extension
+        filename, _ = os.path.splitext(filename)
+        if filename in extra_files:
+            raise ValueError(f"already have config in the extra files with same file name: {filename}.")
```

Comment:
```suggestion
            raise ValueError(f"Base filename '{filename}' is given multiple times in config file list")
```

The alternative is make sure the filenames are unique but that seems too complicated.

Reply:
Sure, updated in the latest commit.

Thanks.